sdk/tables/azure-data-tables/src/main/java/com/azure/data/tables/TableClient.java

```diff
@@ -0,0 +1,91 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+package com.azure.data.tables;
+
+import com.azure.core.annotation.ServiceClient;
+
+import java.util.List;
+import java.util.Map;
+
+@ServiceClient(
+    builder = TableClientBuilder.class)
+public class TableClient {
+    String tableName;
+
+    TableClient(String tableName) {
+        this.tableName = tableName;
+    }
+
+    /**
+     * Queries and returns entities in the given table using the select and filter strings
+     *
+     * @param selectString odata select string
+     * @param filterString odata filter string
+     * @return a list of the tables that fit the query
+     */
+    public List<TableEntity> queryEntity(String selectString, String filterString) {
+        return null;
+    }
+
+    /**
+     * insert a TableEntity with the given properties and return that TableEntity
+     *
+     * @param row                   the RowKey
+     * @param partition             the PartitionKey
+     * @param tableEntityProperties a map of properties for the TableEntity
+     * @return the created TableEntity
+     */
+    public TableEntity insertEntity(String row, String partition, Map<String, Object> tableEntityProperties) {
```

Comment:
We should probably be consistent with the service and call these params `rowKey` and `partitionKey` everywhere.

Reply:
Did we still want to resolve this comment?